package.json

```diff
@@ -21,7 +21,8 @@
   },
   "main": "./dist/video-js/video.js",
   "dependencies": {
-    "videojs-swf": "4.5.2"
+    "videojs-swf": "4.5.2",
+    "vtt.js": "git+https://github.com/gkatsev/vtt.js.git#custom-build"
```

Comment:
@marypearson, this is a different comment thread then you were on before. You might want to ignore it.

@gkatsev If we merge this in and release it as is, what happens to people who just bump their version and are using captions?